_docs/_latest/develop/branch-methodology.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Branch Methodology
+order: 003
+---
+
+#  {{ page.title }}
+
+This page describes the branches in the Forseti repo and how they're used
+and managed.
+
+## Branches
+
+### General Availability (GA) Branches
+
+GA branches are the main branches you'll use to inspect the currently stable &
```

Comment:
Replace this paragraph with:

These are the branches that the Forseti Security project team uses to do development, make releases, and build the website.